Fix: 로깅 구현체 또는 slf4j 의존성 추가하지 않을 경우 ClassNotFoundException 발생

## 💻 발생 환경

<!-- 버그가 발생한 환경을 작성해주세요 -->

- OS: Window
- IDE: IntelliJ IDEA (2024.3.1)
- Java 17
- 새로운 프로젝트에 excel-module 라이브러리 사용
```
api 'io.github.hee9841:excel-module:0.0.0'
```

## 📋 버그 내용

<!-- 버그에 대한 내용을 작성해주세요 -->
<!-- 예시: 로그인 시도 시, http 400 응답과 함께 로그인이 되지 않습니다. -->

- slf4j-api 의존성을 추가하지 않거나, 구현체 의존성을 추가하지 않을 경우 ClassNotFoundException이 발생한다.


#### 😇 원래 예상한 동작

<!-- 버그가 없다고 가정했을때, 예상했던 동작을 작성해주세요 -->
<!-- 예시: 로그인 API인 /api/v1/auth/login에 GET 요청시, http 200 응답과 함께 로그인이 되어야 합니다. -->

- slf4j-api 또는 slf4j 구현체 의존성을 추가하지 않아도 작동되어야 한다.

#### 😈 실제 발생 화면 또는 로그

<!-- 버그가 발생한 화면 또는 로그를 첨부해주세요 -->
<!-- 예시: 로그인 시도 시, 콘솔에 에러 로그가 출력됩니다. -->

```
Exception in thread "main" java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
	at io.github.hee9841.excel.core.SXSSFExcelFile.<clinit>(SXSSFExcelFile.java:39)
	at my.example.Main.main(Main.java:31)
Caused by: java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 2 more
```

#### 🤡 재현 방법

<!-- 버그를 재현하기 위한 방법을 작성해주세요 -->
<!-- 예시: 로그인 API인 /api/v1/auth/login에 GET 요청시, http 400 응답과 함께 로그인이 되지 않습니다. -->

- `@Excel`와  `@ExcelColumn`을 이용해 엑셀파일로 만들  DTO 객체를 지정하고, 엑셀 파일로 만들기 위해 `ExcelExporter`를 build한다.
```
List<MyDto> list = new ArrayList<>();

list.add(new MyDto("d"));

ExcelExporter<MyDto> exporter = ExcelExporter.builder(MyDto.class, list)
    .build();

```

--- a/build.gradle
+++ b/build.gradle
@@ -42,9 +42,8 @@ dependencies {
     implementation 'org.apache.poi:poi-ooxml:5.4.0'
 
     //logging
-    compileOnly 'org.slf4j:slf4j-api:2.0.17'
-    testImplementation 'ch.qos.logback:logback-classic:1.3.15'  //for java 8
-
+    implementation 'org.slf4j:slf4j-api:2.0.17'
+    testImplementation 'ch.qos.logback:logback-classic:1.3.15'
 
     //for annotation processor
     implementation 'com.google.auto.service:auto-service:1.1.1'

